Cargo.toml

```diff
@@ -6,6 +6,12 @@ license = "MIT"
 repository = "https://lab.themackabu.dev/self/pmc"
 description = "PMC is a simple and easy to use PM2 alternative"
 
+[profile.release]
+lto = true
+codegen-units = 1
+opt-level = "z"
```

Comment:
I personally had build issues with codegen level 1, not sure if I should merge this in with this PR.

Reply:
I see